owl/SOMA-ELAN.owl

```diff
@@ -54,6 +55,154 @@
+    </owl:DatatypeProperty>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Classes
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-ELAN.owl#HumanActivityRecording -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ELAN.owl#HumanActivityRecording">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#RecordedEpisode"/>
+        <rdfs:comment>An episode in which one or more human beings perform an activity and are recorded doing so.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForBreakfast -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForBreakfast">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForMeal"/>
+        <rdfs:comment>Describes a goal to prepare a table or dining area for a breakfast.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForDinner -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForDinner">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForMeal"/>
+        <rdfs:comment>Describes a goal to prepare a table or dining area for a dinner.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForLunch -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForLunch">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForMeal"/>
+        <rdfs:comment>Describes a goal to prepare a table or dining area for a lunch.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForMeal -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ELAN.owl#PrepareForMeal">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Goal"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA-ELAN.owl#expectedParticipants"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#integer"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA-ELAN.owl#isFormalEvent"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#boolean"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>A description of a goal to prepare for serving a meal by setting a table.
```

Comment:
Why not just "A description of a goal to set a table?" I do not get why is "for serving" special?

Reply:
In this case pragmatics. This PR is intended to assist the H-area in connecting their NEEMs to SOMA, hence I want to explicitly keep in mind their interests.